contentcuration/contentcuration/viewsets/contentnode.py

```diff
@@ -283,6 +284,10 @@ class ExtraFieldsOptionsSerializer(JSONFieldDictSerializer):
     modality = ChoiceField(choices=(("QUIZ", "Quiz"),), allow_null=True, required=False)
     completion_criteria = CompletionCriteriaSerializer(required=False)
 
+    def update(self, instance, validated_data):
+        validated_data = validate_and_conform_to_schema_threshold_none(validated_data)
```

Comment:
With my suggestion above, this would then happen in the `CompletionCriteriaSerializer` update method instead, so this is only ever called when the completion_criteria have been passed.